scenes/instrument_menu/instrument_menu.gd

```diff
@@ -1,7 +1,18 @@
 extends Control
 
+@onready var instrument_title = %InstrumentTitle
+@onready var instrument_box = %InstrumentBox
+@onready var songs_number = %SongsNumber
+@onready var courses_number = %CoursesNumber
+
+
 func _ready():
-	pass
+	var instrument_data := PlayerVariables.gameplay_instrument_data
+	instrument_box.instrument_data = instrument_data
+	instrument_box.info_visible = false
+	songs_number.text = "%d" % InstrumentDetails.get_songs_number(instrument_data)
+	courses_number.text = "%d" % InstrumentDetails.get_courses_number(instrument_data)
```

Comment:
Not a fan of this way of writing integer to string conversion, I'd prefer str(InstrumentDetails.get_songs_number(instrument_data)) because it's more descriptive of what's happening.